admin/create-theme/theme-styles.php

```diff
@@ -46,7 +60,8 @@ public static function update_style_css( $style_css, $theme ) {
 
 		$css_metadata .= "Text Domain: {$text_domain}
 Tags: {$tags}
-*/
+
+{$copyright}*/
 
```

Comment:
I think we can delete this blank line. Otherwise, a double newline will be inserted at the end of the file.

![double-line](https://github.com/WordPress/create-block-theme/assets/54422211/0455048b-3ffa-4177-856c-7b757e63c14c)

Reply:
Refactored slightly to ensure the blank lines here are as expected.